javascript/javascript2/week3/game/main.js

```diff
@@ -0,0 +1,60 @@
+//html
+const buttonStart = document.querySelector('start-button')
+const leftFild = document.getElementById('left-fild')
+const rightFild = document.getElementById('right-fild')
+
+const inputField = document.getElementById('timeOfGame')
+const infoField = document.querySelector('#info')
+
+const startButton = document.querySelector('#start-button')
+const pressL = document.querySelector('#l')
+const pressS = document.querySelector('#s')
+
+//count
+let countL = 0
+let countS = 0
+
+const keyUpListener = function (event) {
+  if (event.key.toLowerCase() === 's') {
+    countS++
+    pressS.innerHTML = countS
+    infoField.innerHTML = ''
+  } else if (event.key.toLowerCase() === 'l') {
+    countL++
+    pressL.innerHTML = countL
+    infoField.innerHTML = ''
+  } else {
+    infoField.innerHTML = 'Press either S or L'
+  }
+}
+
+startButton.onclick = function (event) {
```

Comment:
Just leave empty brackets when declaring a function if you're not going to pass any argument:)